src/sc_scda.h

```diff
@@ -394,8 +391,6 @@ sc_scda_fcontext_t *sc_scda_fwrite_inline (sc_scda_fcontext_t * fc,
  * All parameters except of \b block_data are collective.
  *
  * This function returns NULL on MPI I/O errors.
```

Comment:
Very strictly speaking, should we just write "on I/O errors" to account for the non-MPI case?

Reply:
Yes, this is more accurate.